libs/usb-drive/src/format_fat32.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+set -euo pipefail
+
+usage () {
+    echo 'Usage: format.sh <device> <label>'
+    exit 1
+}
+
+ROOT_DEVICE_REGEX=^/dev/sd[a-z]$
+
+DEVICE=$1
+LABEL=$2
```

Comment:
Also, for a minor quality of life improvement:

```suggestion
DEVICE=${1:-}
LABEL=${2:-}

if [[ -z $DEVICE || -z $LABEL ]]; then
    usage
fi
```